### PR TITLE
Refactor app structure and style

### PR DESCRIPTION
**Pull request by pfletcherhill**

### CODE REVIEW

File: app/components/Auction.js [hunk 1]
```diff
@@ -1,232 +1,41 @@
...
-      bidIncrement,
-      highestBid,
-      highestBidder,
-      endDate,
-      nftMetadata
-    } = this.auction;
-
-    const { creator, description, name, resourceIdentifiers } = nftMetadata;
-
+    const { auction } = this.auctionStore;
```
Comment (mertcelebi) [inline app/components/Auction.js:33]: Is this used?

File: app/components/Auction.js [hunk 2]
```diff
@@ -1,232 +1,41 @@
...
+            <AuctionTimeLeft />
+            <Divider padded />
+            <AuctionHighestBid />
+            <Divider padded />
+            <AuctionYourBid />
+            <Divider padded />
+            <AuctionBidBox />
+          </div>
+        ) : (
+          <div>Loading...</div>
```
Comment (mertcelebi) [inline app/components/Auction.js:49]: Do you wanna add a better indicator of loading?

File: app/components/AuctionBidBox.js [hunk 3]
```diff
@@ -109,177 +76,122 @@ export default class AuctionBidBox extends Component {
 
   @computed
   get downBid() {
-    return this.newBid.minus(this.props.bidIncrement);
+    const { bidIncrement } = this.props.auctionStore.auction;
```
Comment (mertcelebi) [inline app/components/AuctionBidBox.js:51]: Maybe all of these should be in the `auctionStore`

File: app/components/AuctionHighestBid.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import { inject, observer } from "mobx-react";
+import { computed } from "mobx";
+import BigNumber from "bignumber.js";
+import { colors } from "../styles";
+import styled from "react-emotion";
+
+const Title = styled("div")`
```
Comment (mertcelebi) [inline app/components/AuctionHighestBid.js]: These two seem to be shared a couple times between component. Maybe put into a shared `auction.js` file

File: app/components/AuctionBidBox.js (hunk 3 above)
Comment (mertcelebi) [reply app/components/AuctionBidBox.js:51]: In that case, we also wouldn't need to inject `this.props.store` probably.

File: app/components/AuctionInfo.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  `;
+  return pulse;
+}
+
+const Status = styled("div")`
+  font-size: 15px;
+  font-weight: 400;
+  text-transform: uppercase;
+  display: inline-block;
+  color: ${colors.green};
```
Comment (mertcelebi) [inline app/components/AuctionInfo.js]: Is this `green` for all statuses? Probably should accept a color prop

File: app/components/AuctionTimeLeft.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import { inject, observer } from "mobx-react";
+import CountDown from "./CountDown";
+
+@inject("auctionStore")
+@inject("store")
```
Comment (mertcelebi) [inline app/components/AuctionTimeLeft.js]: Not needed, right?

File: app/components/Desktop.js [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+  statusColor(status) {
+    if (status.equals(0)) return colors.green;
+    else if (status.equals(1)) return colors.yellow;
+    else return colors.blue;
+  }
+
+  render() {
+    const { isReady, windows } = this.props.store;
+    if (!isReady) return <div />;
```
Comment (mertcelebi) [inline app/components/Desktop.js]: This should be a loader. Either add a loader or add a TODO

File: app/components/Desktop.js [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+`;
+
+const Container = styled("div")`
+  position: relative;
+`;
+
+@inject("store")
+@observer
+export default class Desktop extends Component {
+  statusText(status) {
```
Comment (mertcelebi) [inline app/components/Desktop.js]: Not used

File: app/components/Desktop.js [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    if (status.equals(0)) return "Live";
+    else if (status.equals(1)) return "Cancelled";
+    else return "Completed";
+  }
+
+  statusColor(status) {
```
Comment (mertcelebi) [inline app/components/Desktop.js]: Same, same

File: app/components/DesktopWindow.js [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+  closeWindow() {
+    this.props.store.removeWindow(this.props.item.key);
+  }
+
+  handleClick() {
+    this.props.store.focusWindow(this.props.item.key);
+  }
+
+  render() {
+    console.log(this.props);
```
Comment (mertcelebi) [inline app/components/DesktopWindow.js]: :bomb:

File: app/styles/colors.js [hunk 11]
```diff
@@ -1,5 +1,5 @@
 export default {
-  blue: "#3366CC",
+  blue: "blue",
```
Comment (mertcelebi) [inline app/styles/colors.js:2]: No need for quotes

File: app/styles/fonts.js [hunk 12]
```diff
@@ -7,9 +9,14 @@ export const sizes = {
...
 
-export const weights = {
+export const fontWeights = {
   lighter: "200",
   light: "300",
   normal: "400",
   bold: "600"
 };
+
+export const Heading = styled("div")`
```
Comment (mertcelebi) [inline app/styles/fonts.js:19]: This probably shouldn't be in fonts

File: app/styles/global.js [hunk 13]
```diff
@@ -6,11 +6,14 @@ injectGlobal`
...
   }
+  div {
+    box-sizing: border-box;
+  }
   body {
-    background-color: black;
     margin: 0;
     padding: 0;
     font-family: ${sansSerif};
+    background-color: #FFEDE8;
```
Comment (mertcelebi) [inline app/styles/global.js]: Maybe toss this to `colors.js`

File: app/styles/layout.js [hunk 14]
```diff
@@ -24,13 +24,8 @@ export const Spacer = styled("div")`
 
 export const Wrapper = styled("div")`
   width: 95%;
-  max-width: ${screenWidth}px;
+  // max-width: ${screenWidth}px;
```
Comment (mertcelebi) [inline app/styles/layout.js]: Hello

**Review by mertcelebi — CHANGES_REQUESTED** [review]
Did an initial pass

**Comment by mertcelebi** [comment]
Also can you run `./node_modules/.bin/eslint ./app` and fix errors?

File: app/components/Auction.js (hunk 1 above)
Comment (pfletcherhill) [reply app/components/Auction.js:33]: Technically, yes, but I can reference `this.auctionStore.auction` below.

File: app/components/Auction.js (hunk 2 above)
Comment (pfletcherhill) [reply app/components/Auction.js:49]: Later on, yeah

File: app/components/AuctionBidBox.js (hunk 3 above)
Comment (pfletcherhill) [reply app/components/AuctionBidBox.js:51]: Eh I like keeping the `newBid` as a `AuctionBidBox` variable, but I'll refactor more stuff into `AuctionStore`

File: app/components/AuctionHighestBid.js (hunk 4 above)
Comment (pfletcherhill) [reply app/components/AuctionHighestBid.js]: Will add to `Auction.js`

File: app/components/AuctionInfo.js (hunk 5 above)
Comment (pfletcherhill) [reply app/components/AuctionInfo.js]: Good call

File: app/components/AuctionTimeLeft.js (hunk 6 above)
Comment (pfletcherhill) [reply app/components/AuctionTimeLeft.js]: Nope

File: app/styles/colors.js (hunk 11 above)
Comment (pfletcherhill) [reply app/styles/colors.js:2]: Need quotes, it's just a javascript object

File: app/styles/fonts.js (hunk 12 above)
Comment (pfletcherhill) [reply app/styles/fonts.js:19]: Fine for now, until we have a better place for it

File: app/styles/global.js (hunk 13 above)
Comment (pfletcherhill) [reply app/styles/global.js]: Ah yeah

File: app/components/Auction.js [hunk 15]
```diff
@@ -1,232 +1,55 @@
...
-      highestBid,
-      highestBidder,
-      endDate,
-      nftMetadata
-    } = this.auction;
-
-    const { creator, description, name, resourceIdentifiers } = nftMetadata;
-
+    const { auction } = this.auctionStore;
+    console.log("Render Auction", this.auctionStore.auctionId);
```
Comment (mertcelebi) [inline app/components/Auction.js]: Nope

File: app/images/logo-blue.svg [hunk 16]
```diff
@@ -0,0 +1 @@
...3a6.4,6.4,0,0,1-.9-3.6,6.15,6.15,0,0,1,6.3-6.3h30.6c4.8,0,7.5,3,9.3,5.7l44.1,64.8,44.4-64.8c1.8-2.7,4.5-5.7,9.3-5.7h29.1a6.15,6.15,0,0,1,6.3,6.3,5.24,5.24,0,0,1-1.2,3.6l-62.7,93,67.2,97.2a8.53,8.53,0,0,1,1.2,3.6,6.34,6.34,0,0,1-6.3,6.3h-31.8c-4.8,0-7.5-3-9-5.4l-48-67.8-47.1,67.8c-1.8,2.4-4.5,5.4-9.3,5.4H97.89A6.15,6.15,0,0,1,91.59,280.41Z" transform="translate(-2.06 -1.8)"/><path class="cls-1" d="M336.69,279.51V83.91a6.93,6.93,0,0,1,7.2-7.2h27.9a6.93,6.93,0,0,1,7.2,7.2V251h95.1a7.09,7.09,0,0,1,7.5,7.5v21c0,4.2-3,7.2-7.5,7.2H343.89A6.93,6.93,0,0,1,336.69,279.51Z" transform="translate(-2.06 -1.8)"/><path class="cls-1" d="M542.18,279.51V84.21c0-4.5,3-7.5,7.2-7.5h23.1a9.75,9.75,0,0,1,8.7,4.8l86.41,134.4V84.21c0-4.5,3-7.5,7.2-7.5H700a7.25,7.25,0,0,1,7.5,7.5v195a7.18,7.18,0,0,1-7.2,7.5h-23.4a10.21,10.21,0,0,1-8.7-4.8l-86.11-131.4v129c0,4.2-3,7.2-7.5,7.2h-25.2A6.93,6.93,0,0,1,542.18,279.51Z" transform="translate(-2.06 -1.8)"/><path class="cls-1" d="M832,279.51V113.61h-55.5a6.93,6.93,0,0,1-7.2-7.2V84.21c0-4.5,3-7.5,7.2-7.5h153a7.25,7.25,0,0,1,7.5,7.5v22.2a7.18,7.18,0,0,1-7.5,7.2h-55.2v165.9a7.18,7.18,0,0,1-7.5,7.2h-27.6A6.93,6.93,0,0,1,832,279.51Z" transform="translate(-2.06 -1.8)"/></svg>
```
Comment (mertcelebi) [inline app/images/logo-blue.svg]: Add `"ensure_newline_at_eof_on_save": true,` to your sublime preferences

**Review by mertcelebi — CHANGES_REQUESTED** [review]
Two quick things

**Review by mertcelebi — APPROVED** [review]

**Pull request merged**